server/test/index.test.js

```diff
@@ -250,3 +251,49 @@ describe('test signup endpoint with all cases ', () => {
     });
   });
 });
+
+describe('test Edit Agent data /users/:iduser  ', () => {
```

Comment:
Great work at tests

but i suggest you to create multiple files for each group of related tests
its easier to read 

this file is about 300 lines